package.json

```diff
@@ -22,7 +22,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint && npm test"
```

Comment:
npm test should be there for pre-commit

Reply:
Correct I will add it in.. I had removed it because I was working on the tests.